src/sentry/web/api.py

```diff
@@ -407,8 +429,7 @@ def _dispatch(self, request, helper, project_id=None, origin=None, *args, **kwar
             if not project:
                 raise APIError('Client must be upgraded for CORS support')
             if not is_valid_origin(origin, project):
-                tsdb.incr(tsdb.models.project_total_received_cors,
-                          project.id)
+                track_outcome(project.organization_id, project.id, None, 'invalid', 'cors')
```

Comment:
Does this not have a request key associated with the event?

Reply:
yes, but we fetch it a few lines down inside an `if`, and I'm not 100% confident I can rearrange these things without subtly breaking something.

Reply:
I guess it does save the trip to verifying a Key. We can nix it for now and always add it back in later.